.rubocop.yml

```diff
@@ -20,6 +20,8 @@ Metrics/ClassLength:
     - 'lib/hyku_addons/engine.rb'
     - 'app/models/nsu_generic_work.rb'
     - 'app/models/nsu_article.rb'
+    - 'app/models/una_archival_item.rb'
```

Comment:
What are the errors that Rubocop is throwing here? The excludes are over 170 lines now, so it feels like we're not excluding everything that we dont want to refactor.

Reply:
`Metrics/ClassLength: Class has too many lines. [102/100]`

Reply:
More specifically the entire file being used the way it is - 170 lines of excludes in a hidden dot file.

Reply:
But i realise that is outside this PR, so ignore this :)